UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -305,7 +310,20 @@
   LockBoxLib|MdeModulePkg/Library/LockBoxNullLib/LockBoxNullLib.inf
 !endif
   FileExplorerLib|MdeModulePkg/Library/FileExplorerLib/FileExplorerLib.inf
+
+!if $(SECURE_BOOT_ENABLE) == TRUE
+  RngLib|MdePkg/Library/BaseRngLibTimerLib/BaseRngLibTimerLib.inf
+  IntrinsicLib|CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
+  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLibCrypto.inf

```

Comment:
This breaks TLS, probably. Perhaps break this out into another block: 'if TLS, use regular instance. Otherwise, use the smaller crypto one'

Reply:
These could be used by measured boot. maybe just move them out SECURE_BOOT_ENABLE block.